client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,25 +44,16 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul>
         <li>
-          <b>Support for Camunda Cloud 1.2</b><br />
-          With this release, we added new BPMN elements supported by Camunda Cloud 1.2.
-          These new elements are message intermediate throw events, message end events, and manual tasks.
+          <b>Support for Camunda Cloud 1.3</b><br />
+          With this release, we added support for Camunda Cloud 1.3.<br />
+          You can now assign User Tasks to specific assignees or to a group of candidates.
+          Both fields can be configured in the assignment section of the properties panel either using a static value or a feel expression.
```

Comment:
Docs only refer to "expressions" so lets do the same (or am I wrong?)

```suggestion
          Both fields can be configured in the assignment section of the properties panel using a static value or an expression.
```

Reply:
I took that from our slack conversation but I checked now and you are correct.